api/v1alpha1/openapi.yaml

```diff
@@ -3507,16 +3551,25 @@ components:
       properties:
         total:
           type: integer
+          format: int64
           description: The total number of devices in the fleet.
+        applicationStatus:
```

Comment:
You may want to make these statuses required, so we do not need to handle nil pointers later.